core-bundle/src/Event/ContaoCoreEvents.php

```diff
@@ -21,6 +21,13 @@ final class ContaoCoreEvents
      */
     public const BACKEND_MENU_BUILD = 'contao.backend_menu_build';
 
+    /**
+     * The contao.frontend_menu_build event is triggered when a frontend navigation module menu is built.
+     *
+     * @see FrontendMenuEvent
+     */
+    public const FRONTEND_MENU_BUILD = 'contao.frontend_menu_build';
```

Comment:
Best practise is to not use string constants any more for events. We did not introduce a string constant for the `FilterPageTypeEvent` either for example.

Reply:
yes, I was wondering. Changed in 427dc02cce71dad9735afb01b927b5542e82f95f.